### DISCUSSION TITLE
Colored text in output window?

### DISCUSSION THREAD

**Discussion by MacSass** [Tensai75/nzb-monkey-go#26] [Ideas]
Hi, I have been enjoying the colored text output in the orignal NZB-Monkey, it helped to quickly identify successful searches / push to SABNZB (text in green) and failed searches (text in red). 
Can this be implemented to nzb-monkey-go too?

**Comment by Tensai75** [comment]
Well, colored text is already imlemented. See the gif on the README page.

**Comment by Tensai75** [comment]
Indeed, the colour implementation under Windows only worked partially. This should be fixed with the latest version:
https://github.com/Tensai75/nzb-monkey-go/releases/tag/v0.1.13

**Comment by MacSass** [comment]
Yes - looking good now - thanks for the update and quick response!

**Comment by MacSass** [comment]
Hi, I have one additional comment - the blue that shows the input provided - being on black background is virtually impossible to read in windows - see sample here:
![grafik](https://github.com/Tensai75/nzb-monkey-go/assets/25532047/3371a905-2a78-4643-8c1e-6da18f927e36)

I would recommend to make that more bright, if you don´t mind. 

Also the green could get a bit more brightness to make it easier to read, although it is not as bad as the blue.

Thanks for your work - MacSass

**Comment by MacSass** [comment]
Would be great if you could light up colors for windows a bit - it is really hard to read ...
Maybe you there is a "LightBlue" instead of "Blue" and a "LightGreen" instead of "Green"? I not have a clue about Go unfortunately.

Thanks!

**Comment by Tensai75** [comment]
The colors are perfectly readable in the standard cmd window with 100% black background:

![image](https://github.com/Tensai75/nzb-monkey-go/assets/34959627/70da0f51-2976-4081-812a-4eed0920cfa6)

So I am not going to change them.

**Comment by MacSass** [comment]
Thanks for the feedback - interesting - need to investigate why they are so dark for me. I´m not aware I would have made any modifications to CMD window from standard - but will check.

**Comment by qsek1** [comment]
i have the same problem. I've not modified the cmd.
Im running Win10 Pro 22H2

![grafik](https://github.com/Tensai75/nzb-monkey-go/assets/26154753/e7a7a6d0-19a6-4445-a5fc-3440c86f1385)


For every version i have to compile my own build with color.FgHiMagenta instead of color.FgBlue, which looks much better:

![grafik](https://github.com/Tensai75/nzb-monkey-go/assets/26154753/8a05f422-2edc-448a-810e-45c39c4e407a)

**Comment by MacSass** [comment]
Thanks for your comment - good to see I´m not alone with that ... mine looks exactly like yours and I could not figure out why it is so dark, although in Tensai75 screenshot they are a bit brighter than in my windows. I have no clue why ... but anyhow I agree that your screen text has much better readability.
Hope Tensai75 might agree to adjust those a bit ... dark blue on black background is really hard to the eye ..

**Comment by Tensai75** [comment]
Ok, will change it to Cyan in next release.

Regards, Tensai

**Reply by MacSass** [reply]
Great - thank you - looking forward to it ...

**Comment by Tensai75** [comment]
https://github.com/Tensai75/nzb-monkey-go/releases/tag/v0.1.14

> ### Changes
> - use cyan instead of blue

Regards, Tensai

**Reply by MacSass** [reply]
Great - much better readability - thanks for the change!
